Rebuilt long beer example. (#410)
w3c/sdw-sosa-ssn: ssn/rdf/examples/readme.md

# Extended SOSA examples

This folder contains long running examples previously contained within the RDF files in this directory. They correspond to the numbered examples in the [Semantic Sensor Network Ontology](https://www.w3.org/TR/vocab-ssn/) recommendation. 

## Tracking Beer Temperature with IBS-TH2

This example is documented in more detail in [ACME-Beer-Example.md](./ACME-Beer-Example.md) 

> An RDF file containing a [example ttl graph file](Beer-Full-IBS-TH2.ttl) corresponding to this example is available.

Reviewer: ```suggestion
An RDF file containing an [example Turtle graph file](Beer-Full-IBS-TH2.ttl) corresponding to this example is available.
```

## Tree height measurement

> An RDF file containing a [example ttl graph file](tree-height.ttl) corresponding to this example is available.

Reviewer: ```suggestion
An RDF file containing an [example Turtle graph file](tree-height.ttl) corresponding to this example is available.
```

## Number of sunspots

> An RDF file containing a [example ttl graph file](sunspots.ttl) corresponding to this example is available.

Reviewer: ```suggestion
An RDF file containing an [example Turtle graph file](sunspots.ttl) corresponding to this example is available.
```

# Extended SOSA examples

> This folder contains long running examples previously contained within the RDF files in this directory correspond to the numbered examples in the [Semantic Sensor Network Ontology](https://www.w3.org/TR/vocab-ssn/) recommendation. 

Reviewer 2: ```suggestion
This folder contains long running examples previously contained within the RDF files in this directory. They correspond to the numbered examples in the [Semantic Sensor Network Ontology](https://www.w3.org/TR/vocab-ssn/) recommendation. 
```

## Tracking Beer Temperature with IBS-TH2

This example is documented in more detail in [ACME-Beer-Example.md](./ACME-Beer-Example.md) 

> An RDF file containing a [example ttl graph file](Beer-Full-IBS-TH2.ttl) corresponding tothis example is available.

Reviewer 2: ```suggestion
An RDF file containing a [example ttl graph file](Beer-Full-IBS-TH2.ttl) corresponding to this example is available.
```

## Tree height measurement

An RDF file containing a [example ttl graph file](tree-height.ttl) corresponding to this example is available.

## Number of sunspots

> An RDF file containing a [example ttl graph file](sunspots.ttl) corresponding tothis example is available.

Reviewer 2: ```suggestion
An RDF file containing a [example ttl graph file](sunspots.ttl) corresponding to this example is available.
```

## Seismograph measuring ground displacement speed

An RDF file containing a [example ttl graph file](seismograph.ttl) corresponding to this example is available.

## iPhone Barometer

An RDF file containing a [example ttl graph file](iphone_barometer-sosa.ttl) corresponding to this example is available.

## Temperature and Humidity at Coal Oil Point Reserve

An RDF file containing a [example ttl graph file](IDEAS.ttl) corresponding to this example is available.

## Electricity consumption of apartment #134

An RDF file containing a [example ttl graph file](apartment-134.ttl) corresponding to this example is available.

## Wind sensor spinning cups

An RDF file containing a [example ttl graph file](spinning-cups.ttl) corresponding to this example is available.

## Complex sensor capabilities &mdash; DHT22

An RDF file containing a [example ttl graph file](dht22.ttl)     corresponding to this example is available.

## DHT22 Deployment

More documentation is located in (DHT22-room145.md)[DHT22-room145.md].

An RDF file containing a [example ttl graph file](dht22-deployment.ttl)  corresponding to this example is available.

## IP68 Smart Sensor

> This example describes the IP68 Smart Sensor that and some of its capabilities and operating ranges. A specific IP68 Smart Sensor observes the air temperature, and its own battery state.</p>

Reviewer 2: ```suggestion
This example describes the IP68 Smart Sensor and some of its capabilities and operating ranges. A specific IP68 Smart Sensor observes the air temperature, and its own battery state.</p>
```

## IP68 Smart Sensor

This example describes the IP68 Smart Sensor and some of its capabilities and operating ranges. A specific IP68 Smart Sensor observes the air temperature, and its own battery state.</p>

> An RDF file containing a [example ttl graph file](ip68.ttl)  corresponding to this example is available..

Reviewer: ```suggestion
An RDF file containing an [example Turtle graph file](ip68.ttl) corresponding to this example is available..
```
